client/src/pages/ProfileSettings/ProfileSettings.tsx

```diff
@@ -0,0 +1,51 @@
+import { Container, Grid, Link } from '@material-ui/core';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { useState } from 'react';
+import Availability from './../../components/ProfileSettings/Availability/Availability';
+import EditProfile from './../../components/ProfileSettings/EditProfile/EditProfile';
+import Payment from './../../components/ProfileSettings/Payment/Payment';
+import ProfilePhoto from './../../components/ProfileSettings/ProfilePhoto/ProfilePhoto';
+import Security from './../../components/ProfileSettings/Security/Security';
+import Settings from './../../components/ProfileSettings/Settings/Settings';
+import useStyles from './useStyles';
+
+export default function ProfileSettings(props: any): JSX.Element {
+  const Options = ['Edit Profile', 'Profile Photo', 'Availability', 'Payment', 'Security', 'Settings'];
+
+  const classes = useStyles();
+
+  const [currentSection, setCurrentSection] = useState(props.match.params.options);
+
+  const handleClick = (section: string) => {
+    setCurrentSection(section);
+    props.history.push(`/profile/${section}`);
+  };
+
+  return (
+    <Grid container className={`${classes.root}`}>
+      <CssBaseline />
+      <Grid className={`${classes.optionItems}`}>
```

Comment:
You should be passing `container` or `item` (or both) props to `grid` components explicitly. At [Material-UI docs](https://material-ui.com/api/grid/) you can find a bunch of other useful props such as `direction` to differentiate between a row or a column, `justifyContent`/`alignItems` to manage [flex-box behaviour](https://css-tricks.com/snippets/css/a-guide-to-flexbox/), or breakpoint-values like `xs` or `lg` for better responsiveness.

Reply:
I've done a complete redo on the skeleton.